js/ui/text_box/ui.text_editor.base.js

```diff
@@ -439,6 +450,42 @@ const TextEditorBase = Editor.inherit({
         this._input().prop('spellcheck', this.option('spellcheck'));
     },
 
+    _renderLabel: function() {
+        const TEXTEDITOR_WITH_BEFORE_BUTTONS_CLASS = 'dx-texteditor-with-before-buttons';
+
+        if(!this.label && this.$element().find('.' + TEXTEDITOR_LABEL_CLASS).length === 1 || this.$element().find('.' + TEXTEDITOR_LABEL_CLASS).length === 2) {
```

Comment:
It seems we need to extract `this.$element().find('.' + TEXTEDITOR_LABEL_CLASS)` as variable